src/container/BooksList.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import Book from '../components/Book';
+import { removeBook } from '../actions/index';
+import CategoryFilter from '../components/CategoryFilter';
+
+function BooksList() {
+  const allBooks = useSelector(state => state.book);
+  const filter = useSelector(state => state.filter);
+  const dispatch = useDispatch();
+  const handleRemoveBook = id => dispatch(removeBook(id));
+
+  return (
+    <div>
+      <h2>Books List</h2>
+      <table>
+        <thead>
+          <tr>
+            <th>Book ID</th>
+            <th>Title</th>
+            <th>Category</th>
+            <th> </th>
+          </tr>
+        </thead>
+        <tbody>
+          {allBooks
+            .filter(book => {
+              if (filter !== 'All') {
+                if (book.category === filter) {
+                  return true;
+                }
+                return false;
+              }
+              return true;
+            })
```

Comment:
This is nice 👍 but according to the project requirement, we need to create a function for this.

Let's implement a conditional method to determine the book that would be displayed based on the filter value in our redux store.

Screenshot from pathwright;
![image](https://user-images.githubusercontent.com/31897434/100337616-ea481a00-2fd7-11eb-8f31-bdf397565482.png)